css/main.css

```diff
@@ -0,0 +1,498 @@
+/* general */
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0 auto;
+  padding: 0;
+  min-height: 100vh;
+  background-color: rgba(236, 236, 236, 0.952);
+  font-family: nyt-imperial, georgia, 'times new roman', times, serif;
+}
+
+main {
+  background-color: white;
+}
+
+/* nav */
+
+.nav {
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style-type: none;
+  background-color: white;
+  justify-content: space-between;
+  padding-top: 5px;
+  border-bottom: 1px solid rgb(219, 206, 206);
+}
+
+.icons {
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.buttons {
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  padding-right: 20px;
+  height: 20px;
+  padding-left: 20px;
+  border-radius: 10px;
+}
+
+.buttons li {
+  margin-right: 5px;
+}
+
+.nav-bttn {
+  background-color: rgb(86, 123, 149);
+  color: white;
+  border-radius: 5px;
+}
+
+.hamb-logo {
+  width: 20px;
+  height: 20px;
+  padding-left: 20px;
+  justify-content: space-between;
+}
+
+.times-logo {
+  width: 200px;
+  height: 20px;
+  padding-left: 100px;
+}
+
+/* main */
+
+.main-container {
+  display: flex;
+  margin: 0 auto;
+  padding: 0;
+}
+
+.left-conatiner {
+  width: 85%;
+  height: auto;
+  display: grid;
+  background-color: white;
+}
+
+.grid-text p {
+  font-family: nyt-franklin, arial, helvetica, sans-serif;
+  font-size: 14px;
+  line-height: 17px;
+  font-weight: 300;
+  color: #000;
+}
+
+section p:hover {
+  color: grey;
+  cursor: pointer;
+}
+
+.footer-container p:hover {
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+.a11 p:hover {
+  text-decoration: underline;
+}
+
+.right-conatiner {
+  width: 15%;
+  padding-right: 50px;
+  padding-top: 1000px;
+}
+
+.right-conatiner .side-container p:hover {
+  text-decoration: underline;
+  cursor: pointer;
+  color: grey;
+}
+
+.title-img {
+  max-width: 990px;
+  height: auto;
+}
+
+header {
+  text-align: center;
+  background-color: white;
+}
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `background-color: white; properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.bg-white{
  background-color: white;
}
```
**Html file**

```
<div class="bg-white">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_